### PR TITLE
Fix design for invitee request

### PR DESCRIPTION
**Pull request by varungu**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @varungu has signed the CLA. Thanks for the pull request!

File: box-share-sdk/src/main/java/com/box/androidsdk/share/fragments/InviteCollaboratorsFragment.java [hunk 1]
```diff
@@ -165,16 +142,43 @@ public void run() {
...
             String[] emailParts = emails.split(",");
             mController.addCollaborations(getFolder(), mSelectedRole, emailParts).addOnCompletedListener(mAddCollaborationsListener);
         }
     }
 
+    private BoxFutureTask.OnCompletedListener<BoxListInvitees> mGetInviteesListener =
+            new BoxFutureTask.OnCompletedListener<BoxListInvitees>() {
+                @Override
+                public void onCompleted(final BoxResponse<BoxListInvitees> response) {
+                    dismissSpinner();
```
Comment (seema-at-box) [inline box-share-sdk/src/main/java/com/box/androidsdk/share/fragments/InviteCollaboratorsFragment.java]: Move to after activity check

Comment (varungu) [inline box-share-sdk/src/main/java/com/box/androidsdk/share/fragments/InviteCollaboratorsFragment.java]: Done.

**Comment by doncung** [comment]
+1

**Pull request merged**